test/fixtures/minio-fixture/src/main/java/org/elasticsearch/test/fixtures/minio/MinioTestContainer.java

```diff
@@ -15,9 +15,10 @@
 
 public final class MinioTestContainer extends DockerEnvironmentAwareTestContainer {
 
-    // NB releases earlier than 2025-05-24 are buggy, see https://github.com/minio/minio/issues/21189, and #127166 for a workaround
-    // However the 2025-05-24 release is also buggy, see https://github.com/minio/minio/issues/21377, and this has no workaround
-    public static final String DOCKER_BASE_IMAGE = "minio/minio:RELEASE.2025-06-13T11-33-47Z";
+    // NB releases earlier than 2025-05-24 are buggy, see https://github.com/minio/minio/issues/21189, and #127166 for a workaround.
+    // However the 2025-05-24 release is also buggy, see https://github.com/minio/minio/issues/21377, and this has no workaround.
```

Comment:
https://github.com/minio/minio/issues/21377 appears to be resolved. Is it mentioned here because it wasn't released in the 2025-07-23 build?

Reply:
No, that should be fixed in 2025-07-23. The specific MinIO version in use has come up in some conversations with customers so I'm keeping track of the known issues in older versions here. We may end up having to run these tests against multiple MinIO versions in which case we'll need this info - clearly we can extract it from history but it's going to be easier if we have a summary here.

Reply:
Okay, sounds good then 👍